public/app/core/components/NestedFolderPicker/NestedFolderList.tsx

```diff
@@ -226,3 +270,29 @@ const getStyles = (theme: GrafanaTheme2) => {
     }),
   };
 };
+
+// TODO: If we like this, move this out... somewhere
+const SKELETON_COUNTS = [
+  // Each array is a set of percentages that the width is divided into for each 'word'
+  [1],
+  [0.3, 0.7],
+  [0.5, 0.5],
+  [0.7, 0.3],
+  [0.3, 0.2, 0.5],
+];
+
+const SKELETON_WIDTHS = [100, 150, 200];
+
+function SkeletonGroup({ index }: { index: number }) {
+  const count = SKELETON_COUNTS[index % SKELETON_COUNTS.length];
+  const size = SKELETON_WIDTHS[index % SKELETON_WIDTHS.length];
+
+  return (
+    <Stack gap={1}>
+      {count.map((percent, index) => {
+        const width = size * percent;
+        return <Skeleton key={index} width={width} />;
+      })}
+    </Stack>
+  );
+}
```

Comment:
i still lean more towards just making each of these a single block with a varied length, but this is fine for now 👍

Reply:
You know, I'll just remove this completely.